Makefile

```diff
@@ -67,7 +67,7 @@ CMD_SOURCES := $(shell find cmd -name main.go)
 TARGETS := $(patsubst cmd/%/main.go,build/${TARGET}/%,$(CMD_SOURCES))
 PKG_TARGETS := $(patsubst cmd/%/main.go,dist/%-${VERSION}-${TARGET}.zip,$(CMD_SOURCES))
 
-build/${TARGET}/%${GO_EXTENSION}: cmd/%/main.go | build/${TARGET}
+build/${TARGET}/%${GO_EXTENSION}: cmd/%/main.go cmd/*.go *.go | build/${TARGET}
```

Comment:
`*` matches files rather than make rules, IIUC, so it might be better to use `%` in case any go files are generated with make directives in the future.